### DISCUSSION TITLE
Feature request: Add npm open package to globals

### DISCUSSION THREAD

**Discussion by pomdtr** [johnlindquist/kit#683] [Ideas]
Since kit is now available in Windows and Linux, a helper to open a link or file cross-platform would be welcome!

**Comment by johnlindquist** [comment]
I'm sorry, I don't quite follow. What do you mean by "npm open package"? Maybe a screenshot would help me understand?

**Reply by johnlindquist** [reply]
Oh, I assume you mean this?

https://www.npmjs.com/package/open

**Reply by pomdtr** [reply]
Yes, sorry I should have added a link in my previous message.
